packages/sdk/src/services/item-service.ts

```diff
@@ -8,6 +8,14 @@ import { SymmetricKey } from '../models/symmetric-key';
 import { getAllPaged, reducePages, resultHasNext } from '../util/paged';
 import Service, { IDEK, IKEK, IKeystoreToken, IPageOptions, IVaultToken } from './service';
 
+export interface IItemListFilterOptions {
```

Comment:
I think these properties need some typedoc comments since it's not immediately clear that `templateIds` would be a comma separated list of ids (even then I'm just guessing that's what it is) and what `sharedWith` actually represents without checking the swagger docs.

My suggestion is:
- change `templateIds` to `string[]` and then do `templateIds.join(',')` before passing it into `ItemApi.itemsGet()`
- Give `classificationNodeNames` the same treatment
- If sharedWith is a list then same there
- Add a brief typedoc `/** */`  description to each  property to clarify what each should be

Reply:
- agree, I will change `templateIds `& `classificationNodeNames `to `string[]`.
- `sharedWith` is a single UUID and its not a list
- agree, it needs typedoc for each property